### DISCUSSION TITLE
Changing combo to an object in the editor doesn't visually update in the correct way with the default skin

### DISCUSSION THREAD

**Discussion by Gabixel** [ppy/osu#20804] [Q&A] [answered]
With the Argon skin, when you toggle the "new combo" option in an object that's in the past of the timeline, this happens:

![image](https://user-images.githubusercontent.com/43073074/196264150-83a28759-c388-4c6e-a21f-108d8ed7ef96.png)

Here's a video:

https://user-images.githubusercontent.com/43073074/196263661-7c891a02-e9bb-4e7b-9a32-cabe078ed8c0.mp4

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/20807
